docs/manifest-releaser.md

```diff
@@ -153,6 +153,12 @@ documented in comments)
   // absence defaults to false and Releases are created as already Published.
   "draft": true
 
+  // when using the `node-workspace` plugin, force the package discovery
+  // to always link local dependencies, even if the SemVer ranges don't match.
+  // can be used to allow breaking version bumps to update during a release.
+  // see Breaking Changes section below
+  // absence defaults to false
+  "always-link-local": true
```

Comment:
cool, I didn't know this feature existed. I think you're right that it should be the default because whenever I have local pkgA crossing a major version boundary, not only does pkgB's dependency version not update, the build fails because it can't find the old version of pkgA.